dev-tools/kestra-devtools/src/kestra-devtools-cli.ts

```diff
@@ -0,0 +1,88 @@
+// Simple CLI entry point.
+// Built to dist/kestra-devtools-cli.cjs with a shebang so it can be executed directly.
+
+import { getWorkingDir } from "./utilities/working-dir";
+import {exportTestReportSummary} from "./tests-reporting/export-test-report-summary";
+import {getPRContext} from "./github-context";
+
+function parseArgs(argv: string[]) {
+    // argv[0] = node, argv[1] = script, rest are args
```

Comment:
```suggestion
function parseArgs([_, __, ...args]: string[]) {
```

should work and avoid you adding a comment

Reply:
we can improve that next week for sure!